File: week6/mario2.py
from cs50 import get_int

def main():
    height = get_height()
    for i in range(1, height + 1):
        print_row(i, height)

def print_row(row, height):
    print(" " * (height - row) + "#" * row + " " + "#" * row)


def get_height():
    while True:
        height = get_int("Height = ? ")
        if 1 <= height <= 8:
            return height

main()

File: week1/mario2/mario2.c
#include<stdio.h>
#include<cs50.h>

void print_row(int row, int height);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height? ");
    }
    while(height < 1 || height > 8);

    for(int i = 0; i < height; i++)
    {
        print_row(i + 1, height);
    }
}
void print_row(int row, int height)
{
    for(int n = 0; n < height-row; n++)
    {
        printf(" ");
    }

    for(int i = 0; i < row; i++)
    {
        printf("#");
    }
    printf("  ");

    for(int i = 0; i < row; i++)
    {
        printf("#");
    }
    printf("\n");
}

